privacy/basecamp-subprocessors/index.md

```diff
@@ -13,10 +13,8 @@ The following is a list of personal data subprocessors we use. These subprocesso
 * [Braintree](https://www.braintreepayments.com/legal/payment-services-agreement-us). Payment processing services.
 * [Customer.io](https://customer.io/gdpr.html). Transactional email service.
 * [Google Cloud Platform](https://cloud.google.com/security/gdpr/resource-center/). Cloud services provider.
-* [HelloSign](https://www.hellosign.com/trust/compliance/gdpr). Electronic signature service.
 * [Help Scout](https://www.helpscout.net/company/legal/gdpr/). Help desk software.
-* [Mailchimp](https://mailchimp.com/gdpr/). Email newsletter service.
+* [Mailchimp](https://mailchimp.com/gdpr/). Transactional email service.
```

Comment:
Double-checking this one—are we using Mailchimp for 1-1 emails in the app?

Reply:
We have a separate mailing list specifically for current Basecamp customers to be used to announce new features and other Basecamp-the-product specific news. It has not been used yet but it does currently exist.